test/asv.linux.conf.json

```diff
@@ -43,7 +43,7 @@
     ],
     "build_command": [
         "python -mpip install -U scikit-build cython",
-        "python setup.py bdist_wheel --dist-dir={build_cache_dir} -- -DCMAKE_CXX_COMPILER=g++ -DSTATIC_LINKING=True -- -j8",
+        "python setup.py bdist_wheel --dist-dir={build_cache_dir} -- -DCMAKE_CXX_COMPILER=g++ -- -j",
         "python -mpip install https://github.com/QISKit/qiskit-terra/archive/master.zip"
```

Comment:
For consistent benchmarking we should be using a fixed version of terra otherwise how do we know whether a performance improvement is caused by a recent change to terra or to aer?

Reply:
As we don't benchmark Terra compilation process, the only problem I could foresee here is Terra having an issue with the no-optimization parameter, or unrolling, and changing the final resulting circuit in an undesirable way.
Anyway, I don't really mind to stick to latest release, as long as Aer `master` branch is compatible with that release, something that we struggled with in the past.

Reply:
The new Qiskit release process seems to be working quite well, so it's true that we haven't had incompatibilities with Terra releases for a long time since then.